### PR TITLE
Add VideoIntelligence GAPIC client for v1 API

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

File: google-cloud-video_intelligence/google-cloud-video_intelligence.gemspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Gem::Specification.new do |gem|
   gem.name          = "google-cloud-video_intelligence"
-  gem.version       = "0.23.0"
+  gem.version       = "0.24.0"
```
Comment (blowmage) [inline google-cloud-video_intelligence/google-cloud-video_intelligence.gemspec]: The [current release](https://github.com/GoogleCloudPlatform/google-cloud-ruby/releases/tag/google-cloud-video_intelligence%2Fv0.24.1) is already `0.24.1`. This should be bumped up to `0.25.0`.

**Comment by quartzmo** [comment]
Failed with:

```sh
Bundler could not find compatible versions for gem
"google-cloud-video_intelligence":
  In Gemfile:
    google-cloud-video_intelligence

    google-cloud was resolved to 0.44.0, which depends on
      google-cloud-video_intelligence (~> 0.23.0)
```

**Comment by geigerj** [comment]
Updated google-cloud dependency, and google-cloud-video_intelligence gemspec/toc.json/Rakefile.

File: google-cloud-video_intelligence/docs/toc.json [hunk 2]
```diff
@@ -6,6 +6,21 @@
       "type": "google/cloud/videointelligence",
       "patterns": ["\/videointelligence", "google\/protobuf", "google\/rpc"],
       "nav": [
+        {
```
Comment (blowmage) [inline google-cloud-video_intelligence/docs/toc.json]: @quartzmo Won't this also have to be added to the `google-cloud/docs/toc.json`?

Comment (quartzmo) [reply google-cloud-video_intelligence/docs/toc.json]: @blowmage Good catch. We need to always remember to look outside the scope of the package directory with these GAPIC updates. 

@geigerj  [This section](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud/docs/toc.json#L587-L618) of `google-cloud/docs/toc.json` should also be updated.

Comment (geigerj) [reply google-cloud-video_intelligence/docs/toc.json]: Updated.

Comment (quartzmo) [reply google-cloud-video_intelligence/docs/toc.json]: @blowmage This is fixed. Anything else, or is this ready to merge?

**Pull request merged**